react-components/src/hooks/use3dScenes.tsx

```diff
@@ -148,6 +189,40 @@ function populateSceneMapWith360Images(
   });
 }
 
+function populateSceneMapWithGroundplanes(
+  sceneGroundPlanes: Array<NodeItem<GroundPlaneProperties>>,
+  sceneGroundPlaneEdges: Array<EdgeItem<Record<string, Record<string, Transformation3d>>>>,
+  scenesMap: Record<Space, Record<ExternalId, SceneData>>
+): void {
+  sceneGroundPlaneEdges.forEach((edge) => {
+    const { space, externalId } = edge.startNode;
+
+    const mappedGroundPlane = sceneGroundPlanes.find(
+      (groundPlane) =>
+        groundPlane.externalId === edge.endNode.externalId &&
+        groundPlane.space === edge.endNode.space
+    );
+
+    if (scenesMap[space]?.[externalId] === undefined) {
+      return;
+    }
+
+    if (mappedGroundPlane === undefined) {
+      return;
+    }
+
+    const groundPlaneEdgeProperties = Object.values(Object.values(edge.properties)[0])[0];
+    const groundPlaneProperties = Object.values(Object.values(mappedGroundPlane)[0])[0];
+
+    const groundPlane: GroundPlaneProperties = {
+      ...groundPlaneProperties,
+      ...groundPlaneEdgeProperties
+    };
```

Comment:
I see that GroundPlaneProperties is a pretty small type - can we explicitly set what properties comes from which object, or is it just that we don't know?

Reply:
Should have used the `GroundPlane` type instead of `GroundPlaneProperties`, as it also includes a `Transformation3d`. Fixed in [fdb1604](https://github.com/cognitedata/reveal/pull/4181/commits/fdb16049a0672a85b1aeaec3b23762a33a8a64ce)